Mathlib/Analysis/SpecificLimits/Basic.lean

```diff
@@ -604,7 +604,7 @@ end ENNReal
 
 
 theorem factorial_tendsto_atTop : Tendsto Nat.factorial atTop atTop :=
-  tendsto_atTop_atTop_of_monotone Nat.monotone_factorial fun n ↦ ⟨n, n.self_le_factorial⟩
+  tendsto_atTop_atTop_of_monotone (fun _ _ ↦ Nat.factorial_le) fun n ↦ ⟨n, n.self_le_factorial⟩
```

Comment:
I would have thought you should just import `monotone_factorial` here; it's very weird to consider `Tendsto` more elementary than `Monotone` IMO.